cpp/tests/io/csv_test.cpp

```diff
@@ -169,7 +169,8 @@ void check_float_column(cudf::column_view const& col_lhs,
 
   CUDF_TEST_EXPECT_COLUMN_PROPERTIES_EQUIVALENT(col_lhs,
                                                 (wrapper<T>{data.begin(), data.end(), validity}));
-  CUDF_EXPECTS(col_lhs.null_count() == 0, "All elements should be valid");
+  CUDF_EXPECTS(col_lhs.null_count() == 0 and col_rhs.null_count() == 0,
+               "All elements should be valid");
```

Comment:
This is one test that was (kind of) checking the null count. With this additional check, it fails when the null count is incorrect.